From fd2056f2de46fa3be308ad47541f6c7ce8483c91 Mon Sep 17 00:00:00 2001
From: Pino Toscano <pino@kde.org>
Date: Wed, 19 Dec 2018 10:24:51 +0100
Subject: Properly pass an empty permission set

Use QFileDevice::Permissions() to represent no permissions, instead of
hardcoding 0.
---
 src/tests/imagejobtest.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/tests/imagejobtest.cpp b/src/tests/imagejobtest.cpp
index ee578f8..b387e3b 100644
--- a/src/tests/imagejobtest.cpp
+++ b/src/tests/imagejobtest.cpp
@@ -99,7 +99,7 @@ void ImageJobTest::testNonexistant() {
 void ImageJobTest::testUnreadable() {
   QTemporaryFile tmpFile;
   QVERIFY(tmpFile.open());
-  QVERIFY(tmpFile.setPermissions(0));
+  QVERIFY(tmpFile.setPermissions(QFileDevice::Permissions()));
   QUrl u = QUrl::fromLocalFile(tmpFile.fileName());
 
   Tellico::ImageJob* job = new Tellico::ImageJob(u);
-- 
cgit v1.1

